spec/std/http/server/server_spec.cr

```diff
@@ -1,5 +1,8 @@
 require "spec"
 require "http/server"
+{% unless flag?(:without_openssl) %}
+require "../../../support/ssl"
+{% end %}
```

Comment:
Perhaps specs should always expect SSL?

Reply:
Maybe... I don't care.

Reply:
Well, I don't understand why specs should sometimes test something but sometimes shouldn't.

Reply:
True. I don't know if there is any specific use case for speccing without openssl.
If removed, it should be removed from http client specs as well.